modules/microshift-install-multus-rpm.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-install-rpm.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-installing-multus_{context}"]
+== Installing the multiple networks plugin
+
+The {microshift-short} Multus Container Network Interface (CNI) plugin is not installed by default. If you want to attach additional networks to a pod for high-performance network configurations, install the `microshift-multus` RPM package. You can install the plugin for multiple networks during either a new installation or a running cluster. Use this procedure to install the {microshift-short} Multus CNI plugin alongside a new {microshift-short} installation.
+
+[IMPORTANT]
+====
+The Multus RPM cannot be removed after you install it.
+====
+
+.Procedure
+
+. Install the Multus RPM package by running the following command:
++
+[source,terminal]
+----
+$ sudo dnf install microshift-networking-multus
+----
+
+.Next steps
+. Continue with your {microshift-short} installation, including any add-ons.
+
+. After you have finished installing all of the RPMs you want to include, start the {microshift-short} service. The {microshift-short} Multus CNI plugin is automatically deployed.
+
+. Create the custom resources (CRs) needed for your {microshift-short} Multus CNI plugin.
+
+. Configure other networking CNIs as needed.
+
+[TIP]
+====
+If you create your Multus custom resources (CRs) before you complete your installation of {microshift-short}, you can avoid restarting the service to apply them.
+====
+
+//Q: As a user, can I create my Multus CRs while I am doing my initial RHEL/RPM install?
```

Comment:
Yes, NADs can be in RPM or just created in the /etc/microshift/manifests.d and they will be picked up on first boot.
Multus has prefix 000, so they need something bigger like 050 or 100, because we need the CRD first, then CRs